### ISSUE TITLE
503 When deploying to CloudFront

### ISSUE THREAD

**Issue by atrope** [serverless-nextjs/serverless-next.js#1122]
Hello all,

I am getting a strange 503 error when trying to deploy to CF..
on local dev everything works as it should but when we deploy it gives us 503 with the following error on CloudWatch:

```
2021-05-24T22:48:12.720Z	3e59be37-1b27-4eec-aef0-09c729afd8af	ERROR	Invoke Error 	{
    "errorType": "Error",
    "errorMessage": "Cannot find module '../chunks/810.js'\nRequire stack:\n- /var/task/pages/[[...slug]].js\n- /var/task/default-handler-1d169422.js\n- /var/task/index.js\n- /var/runtime/UserFunction.js\n- /var/runtime/index.js",
    "code": "MODULE_NOT_FOUND",
    "requireStack": [
        "/var/task/pages/[[...slug]].js",
        "/var/task/default-handler-1d169422.js",
        "/var/task/index.js",
        "/var/runtime/UserFunction.js",
        "/var/runtime/index.js"
    ],
    "stack": [
        "Error: Cannot find module '../chunks/810.js'",
        "Require stack:",
        "- /var/task/pages/[[...slug]].js",
        "- /var/task/default-handler-1d169422.js",
        "- /var/task/index.js",
        "- /var/runtime/UserFunction.js",
        "- /var/runtime/index.js",
        "    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:815:15)",
        "    at Function.Module._load (internal/modules/cjs/loader.js:667:27)",
        "    at Module.require (internal/modules/cjs/loader.js:887:19)",
        "    at require (internal/modules/cjs/helpers.js:74:18)",
        "    at Object.__webpack_require__.f.require (/var/task/pages/[[...slug]].js:1172:28)",
        "    at /var/task/pages/[[...slug]].js:1097:40",
        "    at Array.reduce (<anonymous>)",
        "    at Function.__webpack_require__.e (/var/task/pages/[[...slug]].js:1096:67)",
        "    at Function.__webpack_require__.x (/var/task/pages/[[...slug]].js:1188:33)",
        "    at /var/task/pages/[[...slug]].js:1205:57"
    ]
}

```

We already tried with latest stable and latest alpha.

Also tried to remove everything and deploy with 

`rm -rf node_modules .next .serverless_nextjs && yarn install && serverless`

but the error is the same.

Thanks!

Next.js version: 10.2.0
Node:  15.6.0

**Comment by dphang** [comment]
This looks very much like https://github.com/serverless-nextjs/serverless-next.js/issues/985, in newer Next.js versions it changed the code splitting logic and `chunks` are output as well. It is already fixed and published in `@sls-next/serverless-component@1.20.0-alpha.4`

If you are already using something on that or greater and it still has issues, can you please compare the `.next/serverless/chunks` directory with the `.serverless_nextjs/default-lambda/chunks` directory and post what it looks like?

**Comment by atrope** [comment]
Hi @dphang,
Again i removed all project and updated to `@sls-next/serverless-component@1.20.0-alpha.7` and now it worked 🥳 

I will be closing this issue!
